src/lib/Theme.ts

```diff
@@ -881,4 +643,306 @@ export default class Theme {
+            let startData = {
+                file_name: 'test.jpg',
+                content_type: 'image/jpeg',
+                size: '1',
+            };
+            let startAssetData = (
+                await UploadService.startUpload(startData, 'application-theme-images')
+            ).data;
+            return (imageCdnUrl = path.dirname(startAssetData.cdn.url));
+        } catch (err) {
+            throw new CommandError(`Failed in getting image cdn base url`);
+        }
+    };
+
+    private static assetCdnBaseUrl = async () => {
+        let assetCdnUrl = '';
+        console.log('path', path.join(process.cwd(), 'theme/assets/fonts'));
+        try {
+            if (fs.existsSync(path.join(process.cwd(), 'theme/assets/fonts'))) {
+                let startData = {
+                    file_name: 'test.ttf',
+                    content_type: 'font/ttf',
+                    size: '10',
+                };
+                let startAssetData = (
+                    await UploadService.startUpload(startData, 'application-theme-assets')
+                ).data;
+
+                return (assetCdnUrl = path.dirname(startAssetData.cdn.url));
+            }
+            return assetCdnUrl;
+        } catch (err) {
+            throw new CommandError(`Failed in getting assets cdn base url`);
+        }
+    };
+
+    private static assetsFontsUploader = async () => {
+        try {
+            if (fs.existsSync(path.join(process.cwd(), Theme.BUILD_FOLDER, 'assets/fonts'))) {
+                const cwd = path.join(process.cwd(), Theme.BUILD_FOLDER, 'assets/fonts');
+                const fonts = glob.sync('**/**.**', { cwd });
+                Logger.warn('Uploading fonts...');
+                await asyncForEach(fonts, async font => {
+                    const assetPath = path.join(Theme.BUILD_FOLDER, 'assets/fonts', font);
+                    await UploadService.uploadFile(assetPath, 'application-theme-assets');
+                });
+            }
+        } catch (err) {
+            throw new CommandError(`Failed to upload assets/fonts`);
+        }
+    };
+
+    private static copyFilesToFdkFolder = async () => {
+        try {
+            Logger.warn('Archiving src...');
+            await fs.copy('./theme', Theme.SRC_FOLDER);
+            fs.copyFileSync('./package.json', Theme.SRC_FOLDER + '/package.json');
+
+            await archiveFolder({
+                srcFolder: Theme.SRC_FOLDER,
+                destFolder: Theme.SRC_ARCHIVE_FOLDER,
+                zipFileName: Theme.ZIP_FILE_NAME,
+            });
+        } catch (err) {
+            throw new CommandError(`Failed to copying theme files to .fdk folder`);
+        }
+    };
+    private static validateAvailableSections = async available_sections => {
+        try {
+            Logger.warn('Validating Files...');
+            available_sections = await Theme.validateSections(available_sections);
+            Theme.createSectionsIndexFile(available_sections);
```

Comment:
This is not related with `validateAvailableSections` function keep this in `themeSync` function itself  @anuragpandey1115